tests/testapps/MinAppCppCx/MinAppCppCx/MinAppCppCx.vcxproj

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You'll want to use PackageReference style nuget deps with the override targets here if you want to use the latest / local packages.